# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/Makefile.am.analysis
# Commit: 3cdfb39376b7
# Full Hash: 3cdfb39376b736597fe658a322389117686b7e82
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 3
# Description:
#   Bug 1892913 - patch 1 - Update cairo source to 1.18.0 r=gfx-reviewers,lsalzman
#   
#   Wholesale replacement of files in gfx/cairo/cairo with contents of the 1.18.0 release tarball,
#   omitting subdirectories that are not part of the core library build.
#   
# ==============================================================================

diff -r 9daa351f875b -r 3cdfb39376b7 gfx/cairo/cairo/src/Makefile.am.analysis
--- a/gfx/cairo/cairo/src/Makefile.am.analysis	Mon May 06 19:22:35 2024 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,35 +0,0 @@
-
-SPARSE = sparse
-sparse:
-	@echo Checking enabled sources with sparse checker
-	@status=true; for f in $(enabled_cairo_sources) $(enabled_cairo_cxx_sources); do \
-		echo $(SPARSE) $(PREPROCESS_ARGS) $(srcdir)/$$f; \
-		$(SPARSE) $(PREPROCESS_ARGS) $(srcdir)/$$f || status=false; \
-	done; $$status
-
-SPLINT = splint -badflag
-splint:
-	@echo Checking enabled sources with splint checker
-	@status=true; for f in $(enabled_cairo_sources) $(enabled_cairo_cxx_sources); do \
-		echo $(SPLINT) $(PREPROCESS_ARGS) $(srcdir)/$$f; \
-		$(SPLINT) $(PREPROCESS_ARGS) $(srcdir)/$$f || status=false; \
-	done; $$status
-
-UNO = uno
-uno:
-	@echo Checking enabled sources with uno checker
-	cd $(srcdir); $(UNO) $(PREPROCESS_ARGS) -DHAVE_CONFIG_H -U__GNUC__ $(enabled_cairo_sources)
-
-headers-standalone: $(enabled_cairo_headers) $(enabled_cairo_private)
-	@echo Checking that enabled public/private headers can be compiled standalone
-	@status=true; for f in $(enabled_cairo_headers) $(enabled_cairo_private); do \
-		echo "  CHECK $$f"; \
-		echo "#include \"$(srcdir)/$$f\"" > headers-standalone-tmp.c; \
-		echo "int main(int argc, char * argv[]) { return 0; }" >> headers-standalone-tmp.c; \
-		$(COMPILE) -o headers-standalone-tmp headers-standalone-tmp.c || status=false; \
-		$(RM) headers-standalone-tmp headers-standalone-tmp.c; \
-	done; $$status
-	@touch $@
-CLEANFILES += headers-standalone
-
-analysis: all headers-standalone sparse splint uno